src/theme.ts

```diff
@@ -8,6 +8,8 @@ const newColors = {
   backgroundError: "#FFE6EA",
   backgroundWarning: "#FFF3E2",
   backgroundSideBar: "#F7F5F5",
+  backgroundBadgeGray: "#e8e7e6",
+  backgroundBadgeDarkGray: "#b2b5b2",
```

Comment:
Do I speak "british" without knowing it? https://www.merriam-webster.com/words-at-play/gray-vs-grey-usage-difference

Reply:
Seems like it :D